api/serializers.py

```diff
@@ -0,0 +1,27 @@
+from client.models import Volunteer, VolunteerSchedule, City
+from rest_framework import serializers
+
+
+class RegistrationSerializer(serializers.HyperlinkedModelSerializer):
+    city = serializers.PrimaryKeyRelatedField(queryset=City.objects.all())
+
+    class Meta:
```

Comment:
Why is this not using the modern class syntax?

Reply:
Also, add optional fields for minors and for terms agreement.

Reply:
> 
> 
> Why is this not using the modern class syntax?

That's how it's like all over the DRF documentation.



> 
> 
> Also, add optional fields for minors and for terms agreement.

Terms agreement would be frontend, I'm adding the minors fields.

Reply:
Possibly a separate issue, but we'd like to mark those who did not approve (yet), with a mechanism for backwards approval. Will be added as a separate issue.